draft-ietf-quic-recovery.md

```diff
@@ -690,9 +690,22 @@ Pseudocode for OnAckReceived and UpdateRtt follow:
     if (sent_packets[ack.largest_acked]):
       latest_rtt = now - sent_packets[ack.largest_acked].time
       UpdateRtt(latest_rtt, ack.ack_delay)
-    // Find all newly acked packets.
+    // Find all newly acked packets and track the smallest
+    smallest_newly_acked = 2^62
     for acked_packet in DetermineNewlyAckedPackets():
```

Comment:
Not an issue of this PR, but I noticed that `DetermineNewlyAckedPackets` is not defined. Do we need to fix that?

Reply:
I can comment what it does more clearly, but I don't want to define it.  There are multiple different ways to determine newly acked packets based on one's data structure choices, so I think it's very implementation dependent.